contracts/FIND.cdc

```diff
@@ -398,7 +401,8 @@ pub contract FIND {
 		}
 
 		pub fun getBalance() : UFix64 {
-			return self.latestBidCallback.borrow()!.getBalance(self.name)
+			let cb = self.latestBidCallback.borrow() ?? panic("The bidder has unlinked the capability")
```

Comment:
add the address of the bidder here, one of the big problems of the default messages is that they do not include this information